corehq/apps/data_interfaces/views.py

```diff
@@ -511,7 +511,7 @@ def get_xform_ids(self, request):
             _request.session = request.session
 
             _request.GET = QueryDict(form_query_string)
-            OTPMiddleware().process_request(_request)
+            OTPMiddleware(lambda req: None)(_request)
```

Comment:
This one's backwards compatible with the old (current) `django-two-factor-auth`, right? Guessing you thought about that, but it just has the look of something that might not be, so I wanted to double check

Reply:
Yes. It's also tested by `corehq.apps.data_interfaces.tests.test_xform_management:XFormManagementTest.test_get_xform_ids__sanity_check`